feature-libs/organization/administration/components/unit/list/toggle-link/toggle-link-cell.component.html

```diff
@@ -1,8 +1,12 @@
 <a
   *ngIf="hasItem"
+  [id]="model.uid"
   class="hide-focus-border"
   [routerLink]="{ cxRoute: route, params: routeModel } | cxUrl"
-  [tabIndex]="tabIndex"
+  (keydown)="onKeydown($event)"
+  [attr.aria-haspopup]="isSwitchable"
+  [attr.data-depth]="model.depthLevel"
+  role="treeitem"
 >
   <button
```

Comment:
I know this is very old code and what you have done avoids the breaking change involved. I'm just not sure why we have a button in an anchor tag at all. I don't mind if you fix this or leave it though since its not in scope.